client/proxy/proxy.go

```diff
@@ -141,6 +146,35 @@ func (pxy *TcpProxy) InWorkConn(conn net.Conn, m *msg.StartWorkConn) {
 		conn, []byte(pxy.clientCfg.Token), m)
 }
 
+// TCP HTTP Tunnel
+type TcpHttpTunnelProxy struct {
```

Comment:
I tried doing that but too much of the current code isn't like this. I'm not going to convert everything now (unless @fatedier thinks strongly otherwise), and I think the code should have a single code style.

Reply:
It can be done after this PR merged.